### ISSUE TITLE
Add ga:isoYear as dimension.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kalnyc67/analytics-issues#321]
```
Name of related component: Core Reporting API

Request summary:

Currently, there is an ISO week dimension, however, using it is quite hard for 
the reasons outlined here:
<https://groups.google.com/d/msg/google-analytics-data-export-api/6Yyodot3Tws/Oc
s-EdqadNMJ>
An example: when requesting data for the whole year 2012, segmented by 
ga:isoWeek, the result row for week 1 will also contain the data for December 
31st, 2012 (which is in 2013-W01). While this isn't technically wrong, it's 
probably also not very useful. Adding the ga:year dimension doesn't help, 
because the whole dataset is inside the year 2012.
Adding ga:isoYear as an available dimension would circumvent this problem as 
the data for 2012-12-31 in the above example would be labeled 2013;01 instead 
of 2012;01.


Use Cases:

Basically the same as for ga:week; being able to consistently report Analytics 
data segmented by ISO week for an arbitrary time range. 


```

Original issue reported on code.google.com by `dorian.k...@webrepublic.ch` on 5 Aug 2013 at 11:49
